src/components/MasterItems.js

```diff
@@ -49,11 +42,22 @@ class MasterItems extends Component {
     this.setState({ entries: [] });
   }
 
+  addComment(entryIndex, e) {
+    const updatedEntry = this.state.entries[entryIndex];
+    updatedEntry.comment = e.target.value;
+    const updatedEntries = R.update(entryIndex, updatedEntry, this.state.entries);
+
+    this.setState({ entries: updatedEntries });
+  }
+
   renderSelectedEntries() {
     return this.state.entries.map((entry, i) => dom.tr(
       { className: 'entries', key: i },
-      dom.td({ className: 'entry-name col-md-4 col-sm-4' }, dom.h3(null, entry.name)),
-      dom.td({ className: 'entry-comment col-md-5 col-sm5' }, dom.h3(null, dom.small(null, entry.comment))),
+      dom.td({ className: 'entry-name col-md-4 col-sm-4' }, dom.h4(null, entry.name)),
+      dom.td({ className: 'entry-comment col-md-5 col-sm5' }, dom.input({
```

Comment:
Should `col-sm5` be `col-sm-5`? Do we need to define both `sm` and `md` if they are the same number of grid columns?